Fuzz: mknote.(*nikonV3).Parse slice out of range

Stack:
```
panic: runtime error: slice bounds out of range [:6] with capacity 2

goroutine 1 [running]:
github.com/simpicapp/goexif/mknote.(*nikonV3).Parse(0x6748e0, 0xc0002ca420, 0x0, 0x0)
	/home/chris/Code/goexif/mknote/mknote.go:58 +0x3d4
github.com/simpicapp/goexif/exif.Decode(0x57a740, 0xc000247e90, 0x2, 0x303, 0x643660)
	/home/chris/Code/goexif/exif/exif.go:304 +0x6fa
github.com/simpicapp/goexif/fuzz.Fuzz(0x7fe967da4000, 0x6b8, 0x6b8, 0x3)
	/home/chris/Code/goexif/fuzz/fuzz.go:11 +0x1a8
go-fuzz-dep.Main(0xc000084f70, 0x1, 0x1)
	go-fuzz-dep/main.go:36 +0x1ad
main.main()
	github.com/simpicapp/goexif/fuzz/go.fuzz.main/main.go:15 +0x52
exit status 2
```
Giant test case:
```
	"MM\x00*\x00\x00\x00\b\x00\v\x01\x0f\x00\x02\x00\x00\x00\x06\x00\x00" +
	"\x00\x92\x01\x10\x00\x02\x00\x00\x00\t\x00\x00\x00\x98\x01\x12\x00\x03\x00\x00" +
	"\x00\x01\x00\x01\x00\x00\x01\x1a\x00\x05\x00\x00\x00\x01\x00\x00\x00\xa2\x01\x1b" +
	"\x00\x05\x00\x00\x00\x01\x00\x00\x00\xaa\x01(\x00\x03\x00\x00\x00\x01\x00\x02" +
	"\x00\x00\x011\x00\x02\x00\x00\x00\x05\x00\x00\x00\xb2\x012\x00\x02\x00\x00" +
	"\x00\x14\x00\x00\x00\xb8\x02\x13\x00\x03\x00\x00\x00\x01\x00\x01\x00\x00\x87i" +
	"\x00\x04\x00\x00\x00\x01\x00\x00\x00̈%\x00\x04\x00\x00\x00\x01\x00\x00" +
	"\x06\xb2\x00\x00\x00\x00Apple\x00iPhone 7" +
	"\x00\x00\x00\x00\x00H\x00\x00\x00\x01\x00\x00\x00H\x00\x00\x00\x0111" +
	".3\x00\x002018:04:03 07:54" +
	":33\x00\x00 \x82\x9a\x00\x05\x00\x00\x00\x01\x00\x00\x02R\x82\x9d" +
	"\x00\x05\x00\x00\x00\x01\x00\x00\x02Z\x88\"\x00\x03\x00\x00\x00\x01\x00\x02" +
	"\x00\x00\x88'\x00\x03\x00\x00\x00\x01\x00P\x00\x00\x90\x00\x00\a\x00\x00" +
	"\x00\x040221\x90\x03\x00\x02\x00\x00\x00\x14\x00\x00\x02b\x90\x04" +
	"\x00\x02\x00\x00\x00\x14\x00\x00\x02v\x91\x01\x00\a\x00\x00\x00\x04\x01\x02" +
	"\x03\x00\x92\x01\x00\n\x00\x00\x00\x01\x00\x00\x02\x8a\x92\x02\x00\x05\x00\x00" +
	"\x00\x01\x00\x00\x02\x92\x92\x03\x00\n\x00\x00\x00\x01\x00\x00\x02\x9a\x92\x04" +
	"\x00\n\x00\x00\x00\x01\x00\x00\x02\xa2\x92\a\x00\x03\x00\x00\x00\x01\x00\x05" +
	"\x00\x00\x92\t\x00\x03\x00\x00\x00\x01\x00\x10\x00\x00\x92\n\x00\x05\x00\x00" +
	"\x00\x01\x00\x00\x02\xaa\x92\x14\x00\x03\x00\x00\x00\x04\x00\x00\x02\xb2\x92|" +
	"\x00\a\x00\x00\x00\x02\x00\x00\x02\xba\x92\x91\x00\x02\x00\x00\x00\x0485" +
	"2\x00\x92\x92\x00\x02\x00\x00\x00\x04852\x00\xa0\x00\x00\a\x00\x00" +
	"\x00\x040100\xa0\x01\x00\x03\x00\x00\x00\x01\xff\xff\x00\x00\xa0\x02" +
	"\x00\x04\x00\x00\x00\x01\x00\x00\x0f\xc0\xa0\x03\x00\x04\x00\x00\x00\x01\x00\x00" +
	"\vТ\x17\x00\x03\x00\x00\x00\x01\x00\x02\x00\x00\xa3\x01\x00\a\x00\x00" +
	"\x00\x01\x01\x00\x00\x00\xa4\x02\x00\x03\x00\x00\x00\x01\x00\x00\x00\x00\xa4\x03" +
	"\x00\x03\x00\x00\x00\x01\x00\x00\x00\x00\xa4\x05\x00\x03\x00\x00\x00\x01\x00\x1c" +
	"\x00\x00\xa4\x06\x00\x03\x00\x00\x00\x01\x00\x00\x00\x00\xa42\x00\x05\x00\x00" +
	"\x00\x04\x00\x00\x06j\xa43\x00\x02\x00\x00\x00\x06\x00\x00\x06\x8a\xa44" +
	"\x00\x02\x00\x00\x00\"\x00\x00\x06\x90\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00" +
	"\x00\x0f\x00\x00\x00\t\x00\x00\x00\x052018:04:03" +
	" 07:54:33\x002018:04:03" +
	" 07:54:33\x00\x00\x00\t\x83\x00\x00\x02o\x00\x00" +
	"\bo\x00\x00\x04\xf9\x00\x00\x02\xa6\x00\x00\x02\x03\x00\x00\x00\x00\x00\x00" +
	"\x00\x01\x00\x00\x01\x8f\x00\x00\x00d\a\xdf\x05\xe7\b\xa9\x052Ap" +
	"ple iOS\x00\x00\x01MM\x00\x12\x00\x01\x00\t\x00\x00" +
	"\x00\x01\x00\x00\x00\t\x00\x02\x00\a\x00\x00\x02.\x00\x00\x00\xec\x00\x03" +
	"\x00\a\x00\x00\x00h\x00\x00\x03\x1a\x00\x04\x00\t\x00\x00\x00\x01\x00\x00" +
	"\x00\x01\x00\x05\x00\t\x00\x00\x00\x01\x00\x00\x00\xb1\x00\x06\x00\t\x00\x00" +
	"\x00\x01\x00\x00\x00\xb9\x00\a\x00\t\x00\x00\x00\x01\x00\x00\x00\x01\x00\b" +
	"\x00\n\x00\x00\x00\x03\x00\x00\x03\x82\x00\f\x00\n\x00\x00\x00\x02\x00\x00" +
	"\x03\x9a\x00\r\x00\t\x00\x00\x00\x01\x00\x00\x00\x13\x00\x0e\x00\t\x00\x00" +
	"\x00\x01\x00\x00\x00\x04\x00\x0f\x00\t\x00\x00\x00\x01\x00\x00\x00\x02\x00\x10" +
	"\x00\t\x00\x00\x00\x01\x00\x00\x00\x01\x00\x14\x00\t\x00\x00\x00\x01\x00\x00" +
	"\x00\x05\x00\x17\x00\t\x00\x00\x00\x01\x00\x00\x00\x00\x00\x19\x00\t\x00\x00" +
	"\x00\x01\x00\x00\x00\x00\x00\x1a\x00\x02\x00\x00\x00\x06\x00\x00\x03\xaa\x00\x1f" +
	"\x00\t\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x00bplist" +
	"00O\x11\x02\x00\x90\x01!\x00B\x00&\x00K\x00v\x00d\x00" +
	"R\x00j\x00z\x00\x84\x00\xb1\x00\xd9\x00\xa3\x00#\x00\xb7\x01T\x01" +
	"\xa8\x00\x18\x00\x1d\x000\x00\xbc\x00\xae\x009\x00M\x00^\x00n\x00" +
	"e\x00a\x00X\x00b\x00A\x02\xa2\x00\xe2\x00\x1c\x00\x10\x00\a\x00" +
	"Q\x00\xa8\x01\xd1\x00G\x00H\x00U\x00V\x00f\x00F\x00\x1e\x01" +
	"n\x02#\x00>\x00;\x00F\x00F\x00f\x00^\x01\xc1\x01)\x00" +
	":\x00N\x00]\x00[\x00/\x00\xf2\x01\x97\x025\x009\x006\x00" +
	"8\x00;\x00U\x00\xea\x00\xeb\x01\xf9\x00\xc4\x00_\x00#\x00\x1d\x00" +
	"d\x00\xa9\x02\xc3\x02E\x00G\x00Q\x00O\x00;\x00-\x00h\x00" +
	"\xed\x00\xfa\x00\xc1\x01\xdf\x01\xac\x01\xf0\x00\r\x01\x11\x02[\x02>\x00" +
	"H\x00T\x00d\x004\x00>\x004\x00\x83\x00u\x00J\x01O\x01" +
	"\x80\x010\x02\x83\x01e\x01Q\x014\x00=\x00G\x00T\x004\x00" +
	"S\x00W\x00b\x00\xe6\x00\x02\x01F\x01=\x01!\x02\xd1\x01\xa7\x01" +
	"\x81\x025\x00D\x00J\x00P\x004\x00N\x00V\x00t\x00\xca\x00" +
	",\x01K\x01E\x01.\x02\xdf\x01\xc8\x01\x94\x024\x00:\x00G\x00" +
	"L\x00>\x00C\x00c\x00q\x00\xae\x00\xd8\x00\xf6\x00\x1a\x01\xef\x01" +
	"\xe1\x01\xc1\x01M\x015\x00@\x00C\x00D\x00A\x006\x00h\x00" +
	"\x81\x00\xa2\x00\xab\x00\xc0\x00\xca\x00\xba\x01\xb0\x01\xa7\x010\x02x0" +
	"0^\x00Q\x00R\x00Z\x000\x00d\x00u\x00\x94\x00\xa7\x00\xb0" +
	"\x00\xd8\x00\xc7\x01\xc1\x01\xba\x01m\x02e\x00i\x00\x81\x00v\x00v" +
	"\x00N\x00f\x00s\x00\xa3\x00\xa0\x00\x9b\x00\xd5\x00\xda\x01\xc6\x01\xd0" +
	"\x01\x1f\x02Z\x00k\x00n\x00n\x00\x86\x00\x99\x00\x98\x00\x8c\x00\x1d" +
	"\x01M\x01\xb9\x00\xc9\x00\xcf\x01\x89\x01\xee\x01c\x02L\x00e\x00m" +
	"\x00{\x00z\x00\x96\x00\xa5\x00\xba\x00\x03\x01Y\x01\x0e\x01\xd6\x00\xcb" +
	"\x01:\x01\x03\x02=\x03>\x00H\x00W\x00q\x00\x83\x00\x86\x00\xa9" +
	"\x00\xcf\x00\\\x01$\x01\xf6\x00\"\x01\xc9\x01:\x01\x02\x02-\x03\x00" +
	"\b\x00\x00\x00\x00\x00\x00\x02\x01\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00" +
	"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\fbplist0" +
	"0\xd4\x01\x02\x03\x04\x05\x06\a\bUflagsUval" +
	"ueYtimescaleUepoch\x10\x01" +
	"\x13\x00\x00\xba\xb7\x95y:\xe9\x12;\x9a\xca\x00\x10\x00\b\x11\x17\x1d" +
	"'-/8=\x00\x00\x00\x00\x00\x00\x01\x01\x00\x00\x00\x00\x00\x00\x00" +
	"\t\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00?\xff\xff\xfe" +
	"\xb2\x00\x00\x02k\xff\xff\xfe\xff\x00\x00T\xa4\xff\xff\xea\xda\x00\x00\x19" +
	"-\x00\x00\x00\r\x00\x00\x00@\x00\x00\x00q\x00\x00\x01\x00q82" +
	"5s\x00\x00\x00\x01\x8f\x00\x00\x00d\x00\x00\x01\x8f\x00\x00\x00d\x00" +
	"\x00\x00\t\x00\x00\x00\x05\x00\x00\x00\t\x00\x00\x00\x05Apple" +
	"\x00iPhone 7 back camer" +
	"a 3.99mm f/1.8\x00\x00\x0f\x00\x01\x00"
```

--- a/mknote/mknote.go
+++ b/mknote/mknote.go
@@ -55,7 +55,7 @@ func (_ *nikonV3) Parse(x *exif.Exif) error {
 	m, err := x.Get(exif.MakerNote)
 	if err != nil {
 		return nil
-	} else if bytes.Compare(m.Val[:6], []byte("Nikon\000")) != 0 {
+	} else if len(m.Val) < 6 || bytes.Compare(m.Val[:6], []byte("Nikon\000")) != 0 {
 		return nil
 	}
 

